wct-core/src/main/webapp/jsp/profileH3-scope.jsp

```diff
@@ -24,7 +24,8 @@
   }
 
   function setUserAgentResult(userAgent, contactURL) {
-    $("#userAgentResult").val(userAgent + " " + contactURL + ")");
+    //$("#userAgentResult").val(userAgent + " " + contactURL + ")");
```

Comment:
In general if code is commented out it should just be deleted (or maybe add a comment as to why it's being commented out but still included in the codebase). We can always go back to previous commits to see what it was like before.